docs/python/programming/dynamiccode.rst

```diff
@@ -61,6 +61,9 @@ code-executing
         of the object. If second arg not present then execute it at the 
         top level of the interpreter. 
          
+        When running under MPI, an error in the statement does NOT call
+        MPI_Abort but returns 0.
```

Comment:
should we reference here `ParallelContext.mpiabort_on_error`?

Reply:
I was dithering about that but have moved to "definitely". I would hate for someone to use ```execute1``` in order to catch an error just because they were unaware of ```pc.mpiabort_on_error(0)```